src/main/java/com/cloudbees/jenkins/plugins/BitbucketJobProbe.java

```diff
@@ -24,24 +28,41 @@ public void triggerMatchingJobs(String user, String url, String scm) {
             SecurityContext old = Jenkins.getInstance().getACL().impersonate(ACL.SYSTEM);
             try {
                 URIish remote = new URIish(url);
-                for (AbstractProject<?,?> job : Hudson.getInstance().getAllItems(AbstractProject.class)) {
-                    LOGGER.info("considering candidate job " + job.getName());
-                    BitBucketTrigger trigger = job.getTrigger(BitBucketTrigger.class);
-                    if (trigger!=null) {
-                        if (match(job.getScm(), remote)) {
-                        	trigger.onPost(user);
-                        } else LOGGER.info("job SCM doesn't match remote repo");
-                    } else LOGGER.info("job hasn't BitBucketTrigger set");
+                for (Job<?,?> job : Hudson.getInstance().getAllItems(Job.class)) {
```

Comment:
BTW if you are editing the line anyway, this is a nice time to `s/Hudson/Jenkins/g`.